cromwell-logging/src/test/scala/cromwell/logging/BusinessLoggingActorSpec.scala

```diff
@@ -0,0 +1,71 @@
+package cromwell.logging
```

Comment:
I would request for your IDE scala style configuration to get rid off all styling issues.